core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -433,7 +435,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
     this.consumers.head.commitSync(Map(tp -> new OffsetAndMetadata(5)).asJava)
   }
 
-  @Test(expected = classOf[TopicAuthorizationException])
+  @Test(expected = classOf[KafkaException])
```

Comment:
Because UToP is retriable, it's not simple to make it pop back up. In this case, if the raise it, we end up stuck in the while(true) loop in ConsumerCoordinator.commitOffsetsSync(). Basically we get back in the problematic we raised in https://issues.apache.org/jira/browse/KAFKA-3727 where we didn't reach a consensus. 
I'll be inclined to keep it like this for now and try to deal with it in 3727.